nautobot/dcim/tests/test_api.py

```diff
@@ -695,6 +696,35 @@ def setUpTestData(cls):
         ]
 
 
+class HardwareFamilyTest(APIViewTestCases.APIViewTestCase):
```

Comment:
Probably also should augment the data in `DeviceTypeTest` to include some non-null `hardware_family` values?